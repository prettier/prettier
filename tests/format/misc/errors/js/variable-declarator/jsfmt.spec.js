run_spec(__dirname, ["babel", "espree", "meriyah"]);
