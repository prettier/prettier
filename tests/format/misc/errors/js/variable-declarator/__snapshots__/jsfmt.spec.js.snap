// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`invalid-const.js [babel] format 1`] = `
"'Const declarations' require an initialization value. (1:10)
> 1 | const foo;
    |          ^
  2 |"
`;

exports[`invalid-const.js [espree] format 1`] = `
"Unexpected token ; (1:10)
> 1 | const foo;
    |          ^
  2 |"
`;

exports[`invalid-const.js [meriyah] format 1`] = `
"[1:10]: Missing initializer in const declaration (1:10)
> 1 | const foo;
    |          ^
  2 |"
`;
